enhancements/single-node-production-edge-cluster-profile.md

```diff
@@ -0,0 +1,380 @@
+
+We are seeing an emerging pattern in which some infrastructure providers and application
+owners desire:
+
+1. A consistent deployment approach for their workloads across these
+   disparate environments.
+3. That the edge sites can operate independantly from the central
+   management hub.
+
+And so, these users who have adopted Kubernetes at their their central
+management sites wish to have independent Kubernetes clusters at the
+more remote sites. 
+
+Of the several options explored for supporting the use of Kubernetes
+patterns for managing workloads at these sites (see the alternatives
+listed blow) a single-node deployment profile of OpenShift is the best
+way to give users a consistent experience across all of their sites.
+
+### Radio Access Network (RAN) Use Case
+
+In the context of telcocommunications service providers' 5G Radio Access
+Networks, it is increasingly common to see "cloud native" implementations
+of the 5G Distributed Unit (DU) component. Due to latency constraints,
+this DU component needs to be deployed very close to the radio antenna for
+which it is responsible. In practice, this can mean running this
+component on anything from a single server at the base of a remote cell
+tower or in a datacenter-like environment serving several base stations.
+
+A hypothetical DU example is an unusually resource-intensive workload,
+requiring 20 dedicated cores, 24 GiB of RAM consumed as huge pages,
+multiple SR-IOV NICs carrying several Gbps of traffic each, and
+specialized accelerator devices. The node hosting this workload must
+run a realtime
+kernel, be carefully tuned to ensure low-latency requirements
+can be met, and be configured to support features like Precision Timing
+Protocol (PTP).
+
+One crucial detail of this use case is the "cloud" hosting this workload
+is expected to be "autonomous" such that it can continue operating with
+its existing configuration and running the existing workload, even when
+any centralized management functionality is unavailable.
+
+### Goals
+
+* This enhancement describes an approach for deploying OpenShift in
+  single-node configurations for production use in environments with
+  "reasonably significant" memory, storage, and compute resources.
+* Clusters built using the `single-node-production-edge` profile
+  should pass most Kubernetes and OpenShift conformance end-to-end
+  tests. Any tests that must be skipped due to differences from a
+  multi-node deployment will be documented.
+
+### Non-Goals
+
+* This enhancement does not address single-node deployments in
+  highly-constrained environments such as Internet-of-things devices
+  or personal computers.
+* This enhancement does not address "developer" use cases. See the
+  [single-node-developer-profile](https://github.com/openshift/enhancements/pull/302)
+  enhancement.
+* This enhancement does not address high-availability for single-node
+  deployments.
+* This enhancement does not address in-place upgrades for this first
```

Comment:
A Telco requirement will be to provide a mechanism to upgrade the machine without re-provisioning the infrastructure and applications

Reply:
That does not match the requirements we've been given so far.

Reply:
+1 for browsell - Bandwidth is often constrained in the far edge telco use cases, and needing to re-provision, and re-pull all of the packages is undesirable. If you could re-provision from a local cache you may resolve this issue, however there is also a desire to minimize downtime which would be decreased if you can remove the need to re-provision.